.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```diff
@@ -50,26 +50,23 @@ body:
     id: stylelint-run
     attributes:
       label: 'How did you run Stylelint?'
-      description: 'For example, CLI, Node.js API, PostCSS plugin, or [Stylelint demo](https://stylelint.io/demo/).'
+      description: 'If applicable, provide a [Stylelint demo](https://stylelint.io/demo/).'
```

Comment:
Consistent format for the description, e.g. placeholders are for examples and descriptions for directions.

To encourage demos where applicable as they help us out.